test/raycast/cameraTest.cpp

```diff
@@ -3,17 +3,25 @@
 
 #include "raycast/camera.h"
 
-TEST_CASE("Camera Test") {
+TEST_CASE("Camera Test", "[camera]") {
   auto origin = glm::vec3(0.0f);
   auto direction = glm::vec3(0.0f, 2.0f, 0.0f);
-  Camera camera(origin, direction);
+  Camera camera(origin, direction, 45.0f);
```

Comment:
this is an example of a constructor that will change after we switch to just the default camera constructor and the constructor with a default fov. All other constructors like this in the tests will need to change.